source/src/in/joind/Main.java

```diff
@@ -52,15 +57,15 @@ public void onCreate(Bundle savedInstanceState) {
     protected void initialiseTabs() {
         tabHost = (FragmentTabHost) findViewById(R.id.tabHost);
         tabHost.setup(this, getSupportFragmentManager(), android.R.id.tabcontent);
-        tabHost.addTab(tabHost.newTabSpec("hot").setIndicator("Hot"), EventListFragment.class, null);
-        tabHost.addTab(tabHost.newTabSpec("upcoming").setIndicator("Upcoming"), EventListFragment.class, null);
-        tabHost.addTab(tabHost.newTabSpec("past").setIndicator("Past"), EventListFragment.class, null);
+        tabHost.addTab(tabHost.newTabSpec(TAB_HOT).setIndicator(TAB_HOT), EventListFragment.class, null);
```

Comment:
With the `setIndicator` method calls, do these need to be pulled from `strings.xml`, rather than from the static values?